README.md

```diff
@@ -64,6 +64,20 @@ Be aware of a few limitations:
 - a line that can take one of two branches gets marked as covered even if only one branch is tested
 - currently, code run by Julia's internal interpreter [is not marked as covered](https://github.com/JuliaLang/julia/issues/37059).
 
+### Exclude specific lines or sections from coverage
+
+To exclude specific sections, wrap the section in `COV_EXCL_START` - `COV_EXCL_STOP` blocks:
```

Comment:
In how far are these "blocks"? Maybe "sandwich" the section between such "comments"? Or "surround ... by"? Or "demarcate"

Reply:
`wrap` is pretty obvious to me, but if this is unclear I'll rephrase that.